app/models/calculateFairhold.ts

```diff
@@ -35,13 +35,27 @@ function calculateFairhold(responseData: ResponseData) {
     throw new Error("maintenancePercentage data is missing or empty");
   }
 
+  function assignHouseSize(numberOfBedrooms: number) {
```

Comment:
I'd view this functionality as part of our data validation, which is being handled by zod.

It's acceptable for the API to receive `houseSize` as undefined, but if it does it should map to a number based on the supplied number of rooms.

This approach would maintain the separation between data validation and business logic.